server/sonar-server/src/test/java/org/sonar/server/qualityprofile/ws/QProfilesWsTest.java

```diff
@@ -113,7 +107,7 @@ public void define_controller() {
     assertThat(controller).isNotNull();
     assertThat(controller.path()).isEqualTo(QProfilesWs.API_ENDPOINT);
     assertThat(controller.description()).isNotEmpty();
-    assertThat(controller.actions()).hasSize(23);
+    assertThat(controller.actions()).isNotEmpty();
```

Comment:
still it's interesting: how to ensure that WS will be enabled in production? For sure it has to handled by an integration test. Still to get quicker feedback loop, it can be enforced by a unit test starting the full container (like a medium test or something like ComputeEngineContainerImplTest). 
In the current state I suggest to only rely on an integration test.